packages/common/nbstore/src/impls/cloud/doc-static.ts

```diff
@@ -45,23 +45,28 @@ export class StaticCloudDocStorage extends DocStorageBase<CloudDocStorageOptions
   protected override async getDocSnapshot(
     docId: string
   ): Promise<DocRecord | null> {
-    const arrayBuffer = await this.connection.fetchArrayBuffer(
-      `/api/workspaces/${this.spaceId}/docs/${docId}`,
-      {
-        priority: 'high',
-        headers: {
-          Accept: 'application/octet-stream', // this is necessary for ios native fetch to return arraybuffer
-        },
+    try {
+      const arrayBuffer = await this.connection.fetchArrayBuffer(
+        `/api/workspaces/${this.spaceId}/docs/${docId}`,
```

Comment:
need id conversion

Reply:
It is acceptable not to do id conversion here, and the previous implementation did not do it.